### ISSUE TITLE
Comma separated shortcuts doesn't works

### ISSUE THREAD

**Issue by nrako** [bry4n/backbone-shortcuts#3]
Assigning multiple shortcuts on a single line separated by comma as in keymaster.js doesn't works.

``` javascript
shortcuts: {
  'right, space': 'someMethod'
}
```

I understand you support the scope feature of keymaster.js https://github.com/madrobby/keymaster/#scopes which make it ugly to supports multiple shortcuts and a scope.

``` javascript
shortcuts: {
  // fulgy but could be nice to supports that for backwards compatibility
  'right, space myscope': 'notReallyNeat'
}
```

Maybe the above could be supported for backwards compatibility but a more clean declaration could be promoted :

``` javascript
shortcuts: {
  'right, space': {scope: 'myScope', method: 'someMethod'}
}
```

Let me know your thoughts and if you haven't time to implement it, I'll see if I have some for a fork-pull-request.

**Comment by aeosynth** [comment]
Looks like @bry4n doesn't have time.

**Comment by aeosynth** [comment]
I think it would be less funky to append the scope to the method:

``` javascript
shortcuts: {
  'right, space': 'someMethod myScope'
}
```

**Comment by nrako** [comment]
@aeosynth yup it's obviously a neater solution.

I almost forgot about this issue, if @bry4n brings his insights I'll still be happy to make a pull-request.

**Comment by aeosynth** [comment]
perhaps making a pull request would help @bry4n bring his insights

**Comment by bry4n** [comment]
Sorry! I have been busy. pull requests are welcome :+1: -- I can work on the fix this weekend.

how about this:

``` javascript
shortcuts: {
  "o, enter <myscope>": "someMethod"
}
```

That solution should be easier for me to update the regex pattern.
https://github.com/bry4n/backbone-shortcuts/blob/master/src/backbone.shortcuts.coffee#L33

**Comment by aeosynth** [comment]
i would find that very hard to parse - you have to check for a comma in the middle of a string

IMO my way is easier to read, since the value currently doesn't take a list

**Comment by nrako** [comment]
The question is to decides if having the ability to have a different scope per event is desired. Or if it would rather introduce unnecessary redundancies. Or maybe both could be supported.

IMO and in my use cases it seems that it would introduce redundancies to have the scope defined with the event.

**Comment by christiangenco** [comment]
+1

Is there a working accepted way to do this?
